cleverhans/model_zoo/simple_cnn.py

```diff
@@ -0,0 +1,42 @@
+"""
+A pure TensorFlow implementation of a convolutional neural network.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import functools
+
+import tensorflow as tf
+
+from cleverhans import initializers
+from cleverhans.model import Model
+
+
+class ModelBasicCNN(Model):
```

Comment:
Maybe rename the class SimpleCNN to match the filename?

Reply:
Good catch, I changed the filename to match the existing name to avoid confusing users of the current name.